### PR TITLE
[Xcode11.2] AuthenticationServices updates 

### PR DESCRIPTION
**Pull request by whitneyschmidt**

### CODE REVIEW

File: src/authenticationservices.cs [hunk 1]
```diff
@@ -608,6 +608,8 @@ interface ASAuthorizationSingleSignOnRequest {
 	enum ASAuthorizationAppleIdButtonType : long {
 		SignIn,
 		Continue,
+		[iOS (13, 2), Mac (10, 15), TV (13, 1)]
```
Comment (whitneyschmidt) [inline src/authenticationservices.cs]: The diff file shows `macos(10.15.1)` - do we add the `.1` here too?

Comment (spouliot) [reply src/authenticationservices.cs]: yes but https://github.com/xamarin/xamarin-macios/pull/7315
someone forgot to update/check the wiki ?

Comment (wachaudh) [reply src/authenticationservices.cs]: sorry got merged by @mandel-macaque while I was out of the office. Updating now.

Comment (whitneyschmidt) [reply src/authenticationservices.cs]: Closing this PR 😆

**Pull request closed without merging**